scripts/build_qat_envoy.sh

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+if [ -f /etc/os-release ]; then
+	source /etc/os-release
+else
+	source /usr/lib/os-release
+fi
+
+readonly YQ_BIN="$(realpath ${PWD}/yq)"
+readonly BAZEL_BIN="$(realpath ${PWD}/bazel.sh)"
+readonly SCRIPTS_DIR="$(realpath "$(dirname "$0")")"
+readonly VERSIONS_FILE="${SCRIPTS_DIR}/../versions.yaml"
+readonly OPENSSL_DIR="${SCRIPTS_DIR}/../openssl"
+readonly QAT_ENGINE_DIR="${SCRIPTS_DIR}/../QAT_Engine"
+readonly ENVOY_DIR="${SCRIPTS_DIR}/../envoy"
+readonly QAT_LIB_DIR="${SCRIPTS_DIR}/../QAT_Lib"
+
+source "${SCRIPTS_DIR}/lib.sh"
+
+install_yq() {
```

Comment:
You need `yq` for parsing the YAML configuration, right? It would be better if you used some tool already present in the distributions to make the security footprint smaller. Otherwise you may need to implement things like checksum checking for the downloaded file.

Reply:
ok, it sounds feasible